src/Symfony/Component/ObjectMapper/Tests/ObjectMapperTest.php

```diff
@@ -371,6 +371,9 @@ public static function objectMapperProvider(): iterable
         yield [new ObjectMapper(new ReflectionObjectMapperMetadataFactory(), PropertyAccess::createPropertyAccessor())];
     }
 
+    /**
+     * @group legacy
+     */
     public function testMapInitializesLazyObject()
```

Comment:
Sould this be renamed to `testMapInitializesLazyObjectPriorToPhp84`, then rename the one below to `testMapInitializesLazyObject`? This way in 8.0, we just have to remove the first test case and the second is untouched apart from the annotation

Reply:
We could that. But that should happen on 7.3 probably where #61146 has been merged.

Reply:
Ah yes indeed, never mind then